README.md

```diff
@@ -51,7 +51,7 @@ Morgan is only interested in the raw data (the tree ring measurement files [`.tx
 
 Write a line of code that will create a text file containing a list of the unique tree IDs (no repeats, no extensions) that have at least 5 lines of data. Build it up, one pipe at a time.
 ```
-wc -l *.txt | sort -n | tail -n 25 | head -n 24 | cut -c 6-13 > Tree_ID_List.txt
+wc -l *.txt | sort -n | tail -n 25 | head -n 24 | cut -c 6-13 > Tree_ID_List.txt #this provides chars 6-13 which is the count plus '1301'.  Adjust your cut pipe to fix.
```

Comment:
Just tested this on PC and your answer is correct.  Mac is apparently counting characters differently!